packages/frontend-api/src/Resources/config/graphql-types/ModelType/Product/ProductDecorator.types.yaml

```diff
@@ -65,6 +65,9 @@ ProductDecorator:
                 description: "Brand of product"
             isSellingDenied:
                 type: "Boolean!"
+            isTemporarilyOutOfStock:
```

Comment:
it would be nice to cover this field in tests

Reply:
added :) 5e1bad3998